res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -0,0 +1,2149 @@
+        'shiftParameterRightGroup1Mode': 0x7D,
+        'parameterRightGroup2Mode': 0x31,
+        'shiftParameterRightGroup2Mode': 0x7E,
+        'parameterRightGroup3Mode': 0x32,
+        'shiftParameterRightGroup3Mode': 0x7F,
+        'parameterRightGroup4Mode': 0x33,
+        'shiftParameterRightGroup4Mode': 0x00
+    };
+
+    PioneerDDJSX.illuminationControl = {
+        'loadedDeck': 0x00, //deck 1-4: 0x00 - 0x03
+        'playPauseDeck': 0x0C, //deck 1-4: 0x0C - 0x0F
+        'cueDeck': 0x10, //deck 1-4: 0x10 - 0x13
+        'djAppConnect': 0x09
+    };
+
+    PioneerDDJSX.valueVuMeter = {
+    	'[Channel1]_current': 0,
+        '[Channel2]_current': 0,
+        '[Channel3]_current': 0,
+        '[Channel4]_current': 0,
+        '[Channel1]_enabled': 1,
+        '[Channel2]_enabled': 1,
+        '[Channel3]_enabled': 1,
+        '[Channel4]_enabled': 1
+    };
+
+    // used for pad parameter selection:
+    PioneerDDJSX.selectedLoopParam = [0, 0, 0, 0];
+    PioneerDDJSX.selectedLoopRollParam = [2, 2, 2, 2];
+    PioneerDDJSX.selectedLoopIntervals = [
+        [1 / 4, 1 / 2, 1, 2, 4, 8, 16, 32],
+        [1 / 4, 1 / 2, 1, 2, 4, 8, 16, 32],
+        [1 / 4, 1 / 2, 1, 2, 4, 8, 16, 32],
+        [1 / 4, 1 / 2, 1, 2, 4, 8, 16, 32]
+    ];
+    PioneerDDJSX.selectedLooprollIntervals = [
+        [1 / 16, 1 / 8, 1 / 4, 1 / 2, 1, 2, 4, 8],
+        [1 / 16, 1 / 8, 1 / 4, 1 / 2, 1, 2, 4, 8],
+        [1 / 16, 1 / 8, 1 / 4, 1 / 2, 1, 2, 4, 8],
+        [1 / 16, 1 / 8, 1 / 4, 1 / 2, 1, 2, 4, 8]
+    ];
+    PioneerDDJSX.loopIntervals = [
+        [1 / 4, 1 / 2, 1, 2, 4, 8, 16, 32],
+        [1 / 8, 1 / 4, 1 / 2, 1, 2, 4, 8, 16],
+        [1 / 16, 1 / 8, 1 / 4, 1 / 2, 1, 2, 4, 8],
+        [1 / 32, 1 / 16, 1 / 8, 1 / 4, 1 / 2, 1, 2, 4]
+    ];
+
+    PioneerDDJSX.setNonDeckSoftTakeover(true);
+    PioneerDDJSX.bindNonDeckControlConnections(true);
+    PioneerDDJSX.initDeck("[Channel1]");
+    PioneerDDJSX.initDeck("[Channel2]");
+    PioneerDDJSX.initDeck("[Channel3]");
+    PioneerDDJSX.initDeck("[Channel4]");
+
+    // show Sampler Load configuration file dialog:
+    if (PioneerDDJSX.samplerShowLoadDialog) {
+        script.toggleControl("[Sampler]", "LoadSamplerBank");
+    }
+
+    // set 32 Samplers as default:
+    engine.setValue("[Master]", "num_samplers", 32);
+
+    // switch on Effect Units as default:
+    for (var index in PioneerDDJSX.fxUnitGroups) {
+        if (PioneerDDJSX.fxUnitGroups.hasOwnProperty(index)) {
+            engine.setValue(index, "enabled", true);
+        }
+    }
+
+    // activate vu meter timer for Auto DJ:
+    if (PioneerDDJSX.twinkleVumeterAutodjOn) {
+        PioneerDDJSX.vuMeterTimer = engine.beginTimer(200, "PioneerDDJSX.vuMeterTwinkle()");
+    }
+
+    // initialize autoDJ tick timer:
+    PioneerDDJSX.autoDJTickTimer = 0;
+};
+
+PioneerDDJSX.shutdown = function() {
+    PioneerDDJSX.setNonDeckSoftTakeover(false);
+    PioneerDDJSX.bindNonDeckControlConnections(false);
```

Comment:
I already guessed something like that, it remained here to be save. But I remove these lines now.